### ISSUE TITLE
Reproducing ReMixMatch + DARP + cRT on CIFAR-10

### ISSUE THREAD

**Issue by YUE-FAN** [bbuing9/DARP#5]
Thanks for the great work. I have a question regarding reproducing the results in table 6 in the supplementary material.

I was trying to reproduce the 79.7% bACC from ReMixMatch + DARP + cRT on CIFAR-10 with the imbalance ratio of 150. According to appendix E, I re-initialized the linear classifier and re-trained it for 20 epochs while freezing other parameters. The re-weighting loss I used is the standard re-weighting by frequency (your paper refers to the loss from [24], maybe mistaken?). The classifier was trained on the labeled dataset.

Other hyper-parameters are: SGD with a learning rate 0.1, momentum 0.9, and weight decay 0.0005. The learning rate of SGD decays by 0.01 at the 16th and 18th epoch. 

Despite the random initialization, the best bACC I got is 76.94%. I wonder where I did wrong. Could you please share your implementation details for this experiment?

Thanks!

**Comment by bbuing9** [comment]
Thanks for your interest in our works!

It might be due to the different experimental setups that you have been tried and actually used in the experiments. 

For helping to reproduce our results, the related codes are now updated (train_crt.py & run_crt.sh)

I hope that it will be helpful for your project.

Thanks!
